scim-swagger.yml

```diff
@@ -354,6 +354,40 @@ components:
         application/scim+json:
           schema:
             $ref: '#/components/schemas/OBAccountPaymentServiceProviders'
+    OBParticipants200:
```

Comment:
For each resource, please add a two line description of its purpose to give the reader some context.